dashboard/app/views/levels/_admin.html.haml

```diff
@@ -1,7 +1,7 @@
 - if current_user && (current_user.permission?(UserPermission::LEVELBUILDER) || current_user.permission?(UserPermission::PROJECT_VALIDATOR))
   = render layout: 'shared/extra_links' do
     %strong= @level.name
-    - if current_user.permission?(UserPermission::LEVELBUILDER)
+    - if (current_user.permission?(UserPermission::LEVELBUILDER) || current_user.permission?(UserPermission::PROJECT_VALIDATOR))
```

Comment:
what's this new permission for? It looks like this case is already checked on line 1